### DISCUSSION TITLE
How to use possession mod?

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#50] [Q&A] [answered]

**Comment by winstxnhdw** [comment, marked as answer]
`M3` enemies while you are in `Phantom`.

**Reply by DanielNotCoder** [reply]
what is M3? is it like side mouse button?

**Reply by Totoqoe** [reply]
Middle click

**Reply by DanielNotCoder** [reply]
Thanks :)

**Comment by joep26020** [comment]
> `M3` enemies while you are in `Phantom`.
It is not working.

**Reply by winstxnhdw** [reply]
Make sure you are on the latest commit.

**Reply by joep26020** [reply]
how do I do that

**Reply by winstxnhdw** [reply]
`git pull` or if you face any issues, simple re-clone the repository following the instructions in the README.

**Reply by joep26020** [reply]
git pull? I'm not sure what you mean... sorry I'm not very technology advanced.

**Reply by winstxnhdw** [reply]
Not a problem. You can just delete the repository and set it all up again just like how you did initially.

**Reply by joep26020** [reply]
Ok, will do. But I got an error I never got before.
